extension-manifest-v3/src/pages/onboarding/iframe.js

```diff
@@ -12,22 +12,19 @@
 import { mount, html, store } from 'hybrids';
 import '@ghostery/ui/onboarding';
 
-import { setupIframeSize, closeIframe } from '@ghostery/ui/iframe';
+import { setupIframe, closeIframe } from '@ghostery/ui/iframe';
 
 import TabStats from '/store/tab-stats.js';
 import Options from '/store/options.js';
 
-setupIframeSize();
+setupIframe();
 
 async function close(host, event) {
   await store.set(Options, {
     onboarding: event.type === 'ignore' ? { shownAt: Date.now() } : null,
   });
 
-  // Options notify other contexts about changes, but
```

Comment:
why this was removed? did Safari fix something? if so, what about older versions?

Reply:
No, it was moved to the `closeIframe` general function, so the timeout is triggered always (to prevent situation, when it would be needed, but we forget about it).

https://github.com/ghostery/ghostery-extension/blob/feat-opera-serp-notification/packages/ui/src/utils/iframe.js#L113